.golangci.yaml

```diff
@@ -0,0 +1,24 @@
+linters:
+  disable-all: true
+  enable:
+    - goimports
+    - gofmt
+    - misspell
+    - revive
+    - gosec
+    - prealloc
+run:
+  skip-files:
+    - /zz_generated_
+    - _generated
+  skip-dirs:
+    - generated
+  deadline: 5m
+  tests: true
+  build-tags:
+    - test
+linters-settings:
+  gosec:
+    # https://github.com/securego/gosec#available-rules
+    config:
+      G306: "0644" # Poor file permissions used when writing to a new file
```

Comment:
do we need poor file permission for this repo?

Reply:
Hi @guangbochen, yes, we need that due to some files created with 0644 permission (the default G306 is 0600).